app/code/community/Vindi/Subscription/Model/CreditCard.php

```diff
@@ -37,7 +37,7 @@ public function loadAttributes($info, $data)
 		if ('saved' === $data->getCcChoice()) {
 			$info->setAdditionalInformation('PaymentMethod', $this->_code)
 				->setAdditionalInformation('use_saved_cc', true)
-				->setAdditionalInformation('installments', $data->getCcInstallments());
+				->setAdditionalInformation('installments', $data->getCcInstallments() || 1);
```

Comment:
Aqui não seria assim?

```suggestion
				->setAdditionalInformation('installments', $data->getCcInstallments() ?: 1);
```

Reply:
Faz sentido :thinking: 
Isso me faz pensar que só validamos o caminho feliz.
@victorsinged, acredito que faça sentido validarmos nos testes a quantidade de parcelas da `Bill`